packages/client/src/client4.ts

```diff
@@ -678,12 +675,6 @@ export default class Client4 {
             body: formData,
         };
 
-        if (formData.getBoundary) {
-            request.headers = {
-                'Content-Type': `multipart/form-data; boundary=${formData.getBoundary()}`,
-            };
-        }
```

Comment:
We do not need to set boundary explicity in the content header when using `FormData`. The interface generates unique boundary string and includes it in the content type header when the form is submitted as part of `multipart/form-data`, which is again set by browser when formData is added to the body of the request. see https://developer.mozilla.org/en-US/docs/Web/API/FormData/Using_FormData_Objects